atomic_reactor/build.py

```diff
@@ -198,7 +196,6 @@ def get_built_image_info(self):
         :return dict
         """
         logger.info("getting information about built image '%s'", self.image)
-        self._ensure_is_built()
```

Comment:
get_build_image_info is called from docker_api to get image_id, to create BuildResults,
so I can't get that method asking for something from future :)